string operation
string arm
moveit_msgs/RobotTrajectory[] trajectory
